nni/retiarii/graph.py

```diff
@@ -535,13 +566,14 @@ def __init__(self, graph, node_id, name, operation, _internal=False):
         self.graph: Graph = graph
         self.id: int = node_id
         self.name: str = name or f'_generated_{node_id}'
+        self.python_name: Optional[str] = None
```

Comment:
Suggest adding a comment line to state that " `python_name` is None by default. It should be set after initialization if it is needed."

Reply:
Thanks for the comment. Have fixed it.